code/modules/cm_marines/orbital_cannon.dm

```diff
@@ -141,6 +141,11 @@ var/obj/structure/ship_rail_gun/almayer_rail_gun
 /obj/structure/orbital_cannon/proc/chamber_payload(mob/user)
 	set waitfor = 0
 
+	if(!loaded_tray)
+		if(user)
```

Comment:
User should always exist, if not that's a problem of other pieces of the OB code and should be handled there so you can safely remove this.

Reply:
Can I remove it in the other instances in the OB code without breaking it just to make it more streamline?